src/app/_components/Form/FormCheckbox.tsx

```diff
@@ -1,26 +1,57 @@
 import { Checkbox, type CheckboxProps, Field, Label } from '@headlessui/react'
 import { Check } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
 
 import { Icon } from '@/components/Icon'
 
 type ExcludedHeadlessUIProps = 'className'
 
 export type FormCheckboxProps = {
   label: string
+  labelSize?: 'sm' | 'md'
 } & Omit<CheckboxProps, ExcludedHeadlessUIProps>
 
-export function FormCheckbox({ label, ...rest }: FormCheckboxProps) {
+const TOUCH_TARGET = {
+  base: 'size-5',
+  clickArea: 'p-3.5',
+  clickAreaOffset: '-m-3.5',
+}
```

Comment:
@barbaraperic @CharlyMartin - We need to tackle this naming once and for all 🤣 I don't like these much either.

What about:

```
  visibleElementSize: 'size-5',      
  touchAreaPadding: 'p-3.5',    
  touchAreaOffset: '-m-3.5',
```  
 
 ```
  elementVisibleSize: 'size-5',  
  additionalTouchPadding: 'p-3.5', 
  alignmentOffset: '-m-3.5',       

```

```
  displaySize: 'size-5',    
  expandTouchArea: 'p-3.5',       
  adjustPositionForPadding: '-m-3.5', 
```

Reply:
✅